service_manifest.yml

```diff
@@ -9,7 +9,7 @@ stage: CORE
 category: Static Analysis
 
 file_required: true
-timeout: 150
+timeout: 600
```

Comment:
Wow do we really want Deobfuscator to only pre-empt after 10 minutes? Seems like a long time for a static service

Reply:
Well that's just extra headroom, the longest I've seen a sample take is 6min. This might be a change we'll see for more services, where if we can't optimize the code to handle the requests faster, we'll give it more time.

But it's more about finding that fine line. Alternatively, this could be something deep scanning could take care of.